scripts/pull_image-builder.py

```diff
@@ -5,11 +5,14 @@
 import sys
 import shutil
 import tempfile
+import os
 
+from utils import patch_md
 
 def main():
     root = pathlib.Path(__file__).parent.parent
 
+    base_url = "https://github.com/osbuild/image-builder-cli"
```

Comment:
This question is a bit orthogonal to this PR but why do we need a separate script to parse this repo/set of readmes? (As in: what makes it special?)

Reply:
The were just two scripts :-/
I think the usecase is just different. One is using explicit files mentioned in the `readme-list` file the other is automatically importing all the `.md` files. That's why I moved the common code to `utils.py`